src/log.c

```diff
@@ -133,7 +133,6 @@ void logging_init(void)
 out:
     if (err) {
         loglevel = LOGLEVEL_INFO;
-        ERROR("%s\n", err->message);
```

Comment:
If this line only displays the annoying start up error, i'm fine with this change.
If not, we should discriminate when to display it when to not.